components/price.js

```diff
@@ -20,7 +20,7 @@ export const CURRENCY_SYMBOLS = {
   ZAR: 'R '
 }
 
-const endpoint = (fiat) => `https://api.coinbase.com/v2/prices/BTC-${fiat ?? 'USD'}/spot`
+const endpoint = (fiat) => `https://stacker.news/api/price?fiat=${fiat ?? 'USD'}`
```

Comment:
Next.js seems to only support absolute URLs.

This means that as it is now, this will fetch data from the production API even if the API is run locally for development.

Reply:
Outdated because GraphQL API is now used